site/js/jquery.console.js

```diff
@@ -37,714 +37,809 @@
+
+        function cancelExecution() {
+            if (typeof config.cancelHandle == 'function') {
+                config.cancelHandle();
+            }
+        }
+
+        ////////////////////////////////////////////////////////////////////////
+        // Handle a command
+        function handleCommand() {
+            if (typeof config.commandHandle == 'function') {
+                disableInput();
+                addToHistory(promptText);
+                var text = promptText;
+                if (extern.continuedPrompt) {
+                    if (continuedText)
+                        continuedText += '\n' + promptText;
+                    else continuedText = promptText;
+                } else continuedText = undefined;
+                if (continuedText) text = continuedText;
+                var ret = config.commandHandle(text, function(msgs) {
+                    commandResult(msgs);
+                });
+                if (extern.continuedPrompt && !continuedText)
+                    continuedText = promptText;
+                if (typeof ret == 'boolean') {
+                    if (ret) {
+                        // Command succeeded without a result.
+                        commandResult();
+                    } else {
+                        commandResult(
+                            'Command failed.',
+                            "jquery-console-message-error"
+                        );
+                    }
+                } else if (typeof ret == "string") {
+                    commandResult(ret, "jquery-console-message-success");
+                } else if (typeof ret == 'object' && ret.length) {
+                    commandResult(ret);
+                } else if (extern.continuedPrompt) {
+                    commandResult();
+                }
+            }
+        };
+
+        ////////////////////////////////////////////////////////////////////////
+        // Disable input
+        function disableInput() {
+            acceptInput = false;
+        };
+
+        // Enable input
+        function enableInput() {
+            acceptInput = true;
+        }
+
+        ////////////////////////////////////////////////////////////////////////
+        // Reset the prompt in invalid command
+        function commandResult(msg, className) {
+            column = -1;
+            updatePromptDisplay();
+            if (typeof msg == 'string') {
+                message(msg, className);
+            } else if ($.isArray(msg)) {
+                for (var x in msg) {
+                    var ret = msg[x];
+                    message(ret.msg, ret.className);
+                }
+            } else { // Assume it's a DOM node or jQuery object.
+                inner.append(msg);
+            }
+            newPromptBox();
+        };
+
+        ////////////////////////////////////////////////////////////////////////
+        // Report some message into the console
+        function report(msg, className) {
```

Comment:
:information_source: This change is coming from this commit: https://github.com/chrisdone/jquery-console/commit/c8812e4a2a3a4328775391bff06a77aa42753bd0